examples/js/controls/OrbitControls.js

```diff
@@ -133,12 +133,7 @@ THREE.OrbitControls = function ( object, domElement ) {
 			offset.applyQuaternion( quat );
 
 			// angle from z-axis around y-axis
-
-			theta = Math.atan2( offset.x, offset.z );
-
-			// angle from y-axis
-
-			phi = Math.atan2( Math.sqrt( offset.x * offset.x + offset.z * offset.z ), offset.y );
+			spherical.fromVector3( offset );
```

Comment:
If you want to switch from `atan2()` to `acos()`, we may have to clamp the arg due to roundoff problems, as we do in `Vector3.angleTo()`. I am not sure if that is required here or not...